managed-services/transactional-email/api-cli/send-emails-with-api.mdx

```diff
@@ -0,0 +1,131 @@
+---
+meta:
+  title: Sending an email using the Transactional Email API
+  description: This page explains how to send an email to multiple recipients using the Transactional Email API
+content:
+  h1: Sending an email using the Transactional Email API
+  paragraph: This page explains how to send an email to multiple recipients using the Transactional Email API
+tags: transactional email send multiple-headers send-emails
+dates:
+  validation: 2024-04-24
+  posted: 2024-04-24
+categories:
+  - managed-services
+---
+
+This page shows you how to send a simple transactional email in `JSON` format to multiple recipients using the `additional_headers` parameter using the Scaleway Transactional Email API.
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Configured your API key](/identity-and-access-management/iam/how-to/create-api-keys/)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- [Installed curl](https://curl.se/download.html)
+- [Configured](/managed-services/transactional-email/how-to/configure-domain-with-transactional-email/) your domain name with Transactional Email
+- [Added SPF, DKIM](/managed-services/transactional-email/how-to/add-spf-dkim-records-to-your-domain/), [MX](/managed-services/transactional-email/how-to/add-mx-record-to-your-domain/) and [DMARC](/managed-services/transactional-email/how-to/add-mx-record-to-your-domain/) records to your domain
+
+
+1. Open a terminal and paste the following code to configure your environment variables. Make sure that you add your own values.
+
+        ```
+        export SCW_ACCESS_KEY="<API access key>"
+        export SCW_SECRET_KEY="<API secret key>"
+        export SCW_PROJECT_ID="<Scaleway Project ID>"
+        ```
+2. Run the following command to retrieve your domain's ID, as you will need it in the next step. The output should return your domain's information.
+        ```
+        curl -X GET \
+        -H "X-Auth-Token: $SCW_SECRET_KEY" \
+        "https://api.scaleway.com/transactional-email/v1alpha1/regions/fr-par/domains"
+        ```
+3. Run the following command to query your domain. Make sure that you replace `<domain-id>` with the ID of your domain.
+        ```
+        curl -X POST "https://api.scaleway.com/transactional-email/v1alpha1/regions/$REGION/domains/<domain-id>/check" \
```

Comment:
This query is used to verify an unchecked domain. IDK if it's in the scope of this doc.

Either you need to add the step before (Create the domain via API)

Or remove this step and starts to actually sending an email with the API